packages/api-headless-cms/src/content/plugins/graphqlFields/object.ts

```diff
@@ -29,11 +29,8 @@ const typeFromField = (params: TypeFromFieldParams): TypeFromFieldResponse | nul
     const mTypeName = createManageTypeName(typeName);
 
     // `field` is an "object" field
-    const fields = field.settings.fields as CmsModelField[];
+    const fields: CmsModelField[] = field.settings.fields || [];
 
-    if (!fields || fields.length === 0) {
```

Comment:
Just to confirm - we want to remove this check?

Asking because of the code below:

![image](https://user-images.githubusercontent.com/5121148/156210786-6da4c10a-83db-4269-846c-f298467f6ee7.png)

Looks like now there's a possibility that a type can be included in the schema, without any fields. Previously that wasn't possible because of the check that would return `null` if no fields are defined.

I might be missing something here.